devtools/conda-envs/test_env.yaml

```diff
@@ -22,20 +23,17 @@ dependencies:
   - mdtraj
   - intermol
   - nglview
-  - openff-evaluator
```

Comment:
I found with some `conda-tree` foo that this was bringing in a lot of things that weren't used - in particular Yank, which pulls down the entire Jupyter ecosystem for some reason. It is used in tests and I'd like to bring it back or move the functionality I need out of it.